README.md

```diff
@@ -119,6 +119,39 @@ Once in the web app:
 * Explore citations and sources
 * Click on "settings" to try different options, tweak prompts, etc.
 
+## Automate deployment
+
+You can use GitHub actions or Azure DevOps to deploy this demo. For example, after pushing changes to the repo.
+
+There is a `.github` and `.azdo` folder which provides a pipeline (or workflow) definition (`azure-dev.yml`). These pipelines require some configuration for the provider (GitHub or Azure DevOps), for example, setting secrets or variables. You can use `azd` to create the configuration for the provider:
+
+For `GitHub`, run:
+> azd pipeline config
+
+For `Azure DevOps`, run: 
+> azd pipeline config --provider azdo
+
+Notes:
+
+- Enable the `GitHub workflow` to provision and deploy by renaming the file `azure-dev.yml.disabled` to `azure-dev.yml`.
+
+- Create an entry in `main.paramenters.json` for `openAiResourceGroupLocation` like:
+```json
+  "parameters": {
+    "openAiResourceGroupLocation":
+    {
+      "value": "LOCATION"
+    }
+  }
+```
+Use a valid location for `LOCATION`.
+
+- `pipeline config` checks if there is a git-remote `origin` for the current folder and tries to set up that repository if found. If you cloned this repo from `Azure-Samples/repo-name`, azd will try to set up this repo with secrets/variables and fail (due to access). You can tell `azd` to use a different git-remote by using the `--remote-name` argument like: `azd pipeline --remote-name foo`. This will make `azd` to look for a remote `foo` and if it is not found, it will help you to create a new private repository to host the code and pipelines.
+
+- When using `Azure DevOps`, you will need a personal access token. Follow [this](https://learn.microsoft.com/en-us/azure/developer/azure-developer-cli/configure-devops-pipeline?tabs=GitHub) document for the mininum required roles.
+
+- There is a current limitation for `azd` to ONLY support Azure DevOps repositories using `HTTPS` (`ssh` not yet supported). 
```

Comment:
Is there an FAQ/troubleshooting thing we can link to from here, just to make it easier for us to check when this is no longer the case?